### ISSUE TITLE
Factorization error on 1024×1024 resolution ffhq

### ISSUE THREAD

**Issue by StevenShaw1999** [james-oldfield/PandA#2]

**Comment by james-oldfield** [comment]
Hi Steven, thanks for your interest and for raising the issue!

One issue we note in the paper is that the vanilla gradient descent solution is sensitive to the learning rate. Lowering the learning rate to `1e-7` for 1024x1024 images should do the job (I've just tested this now on the colab notebook).

Alternatively, you can uncomment the line below: `M.decompose_autograd(lr=1e-3, ranks=[512, 16], verbose=True)` to instead use the Adam solver, which is not as sensitive to the learning rate.

Let me know if this helps resolve it for you!

James.

**Comment by StevenShaw1999** [comment]
Ok thank you for your quick reply! I've also realized this issue.
